Makefile

```diff
@@ -56,7 +56,7 @@ endif
 govet:
 ifneq ($(HAS_SERVER),)
 	@echo Running govet
-	@# Workaroung because you can't install binaries without adding them to go.mod 
+	@# Workaroung because you can't install binaries without adding them to go.mod
```

Comment:
*Workaround

Reply:
Yes, this was already there. I would rather change it in a separate PR to keep related things in a single PR. The trailing space got removed because it's an editor setting of mine. Probably I should revert that too. But let's wait for the review to be done.